syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -202,6 +203,9 @@ def sub(self, _self, other):
 
     __sub__ = sub
 
+    def __rsub__(self, other):
```

Comment:
Not exactly because `__rsub__(self, other)` is called after `other - self` failed, typically because other was an integer. So we need to reverse arguments.